utils/fixed_decimal/src/compact.rs

```diff
@@ -26,18 +26,44 @@ pub struct CompactDecimal {
 }
 
 impl CompactDecimal {
+    pub fn noncompact(significand: FixedDecimal) -> Self {
```

Comment:
Nit, here and below: This needs docs.

Also, since this is a constructor, it should say `new` or `from` to make it more discoverable. Maybe:

- `from_significant`
- `new_non_compact`

Reply:
I ended up not needing this constructor anyway; removing it for now.